File: poo/PHP/uberX.php
<?php
require_once("car.php");
class UberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $passegers, $driver, $brand, $model){
        parent::__construct($license, $passegers, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}
?>

File: poo/Java/UberX.java
class UberX extends Car {
    String brand;//Atributos de ubeX nada mas
    String model;

    public UberX(String license, Account driver, String brand, String model) {
        super(license, driver);//Referirse al metodo contructor de la clase padre
        this.brand = brand;
        this.model = model;
    }
}
